### DISCUSSION TITLE
Onefile executable reading the config in exe path

### DISCUSSION THREAD

**Discussion by cordmouse** [MacielG1/VodRecovery#17] [General]
Creating an executable version of the script (windows) with pyinstaller creates the issue of passing the configuration file settings and help. I would prefer bundling of course default settings with the --onefile build, however only copying them from temp directory to the execution path (if not exist) and the script looking for and using the configuration files from the execution directory. 
Can someone help with build instructions for this process?
Below seems to resolve this task but I am unable to implement these changes - basic tools usage and programming knowledge.
https://stackoverflow.com/questions/47850064/add-configuration-file-outside-pyinstaller-onefile-exe-file-into-dist-director

**Comment by MacielG1** [comment]
I don't know much about PyInstaller, but you could try something like this:
`pyinstaller --add-data "config/*:config" --add-data "lib/*:lib" --onefile vod_recovery.py`

**Comment by cordmouse** [comment]
Config files were bundled correctly although they are still in the temp folder and settings need to be adjusted every time the executable is run. Of course settings can be adjusted prior bundling- especially the download path. Thank you.
